projects/packages/identity-crisis/.gitignore

```diff
@@ -0,0 +1,4 @@
+wordpress
+/build
```

Comment:
You might want to add this to the monorepo root's `.phpcs.config.xml` and/or `.eslintignore` (depending on which kinds of files it contains), similar to what I did for connection-ui's build dir recently.

Reply:
:thumbsup: good idea